src/Workspaces/Core/Portable/Shared/Extensions/ISymbolExtensions.cs

```diff
@@ -1156,7 +1156,7 @@ static string BuildXPathForElement(XElement element)
                 var path = "/node()[not(self::overloads)]";
                 for (var current = element; current != null; current = current.Parent)
                 {
-                    var currentName = current.Name;
+                    var currentName = current.Name.ToString();
```

Comment:
📝 Previously, `currentName` had the type `XName`. This resulted in an exception when assigning `"*"` in the implicit conversion from `string`&rarr;`XName` because `*` is not a valid XML name.